modular_skyrat/modules/salon/code/scissors.dm

```diff
@@ -70,8 +72,14 @@
 
 		playsound(target_human, 'modular_skyrat/modules/salon/sound/haircut.ogg', 100)
 
-		if(do_after(user, facial_haircut_duration, target_human))
+		if(do_after(user, (HAS_TRAIT(user, TRAIT_BARBER) ? FACIAL_HAIRCUT_SPEED_BARBER : FACIAL_HAIRCUT_SPEED_NONBARBER), target_human))
 			target_human.facial_hairstyle = facial_hair_id
 			target_human.update_hair()
-			user.visible_message(span_notice("[user] successfully cuts [target_human]'s facial hair!"), span_notice("You successfully cut [target_human]'s facial hair!"))
+			user.visible_message(span_notice("[user] gracefully cuts [target_human]'s facial hair!"), span_notice("You gracefully cut [target_human]'s facial hair!"))
+			user.balloon_alert_to_viewers("snip snippity!")
```

Comment:
against the intended purpose of balloon alerts, they're supposed to be quick and concise ways of telling what's happened/happening